pkg/strategy/bollmaker/strategy.go

```diff
@@ -653,6 +671,36 @@ func (s *Strategy) Run(ctx context.Context, orderExecutor bbgo.OrderExecutor, se
 			return
 		}
 
+		// Update spreads
+		if s.DynamicSpreadWindow > 0 && kline.Direction() == types.DirectionUp {
+			s.DynamicAskSpread.Update(kline.GetHigh().Sub(kline.GetOpen()).Div(kline.GetOpen()).Float64())
+		}
+		if s.DynamicSpreadWindow > 0 && kline.Direction() == types.DirectionDown {
+			s.DynamicBidSpread.Update(kline.GetOpen().Sub(kline.GetLow()).Div(kline.GetOpen()).Float64())
+		}
+		if s.DynamicSpreadWindow > 0 && s.DynamicBidSpread.Length() >= s.DynamicSpreadWindow {
+			dynamicBidSpread := fixedpoint.NewFromFloat(s.DynamicBidSpread.Last())
+			if !s.MaxBidSpread.IsZero() && dynamicBidSpread.Compare(s.MaxBidSpread) > 0 {
+				s.BidSpread = s.MaxBidSpread
```

Comment:
what's the default value of BidSpread and AskSpread when window dynamic bid spread's length is less than the window size?

Reply:
askSpread and bidSpread
Or spread